elastic-apm-agent/src/test/java/co/elastic/apm/agent/premain/ExcludeJvmBootstrapCheckTest.java

```diff
@@ -0,0 +1,85 @@
+package co.elastic.apm.agent.premain;
+
+import org.junit.jupiter.api.Test;
+
+import javax.annotation.Nullable;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ExcludeJvmBootstrapCheckTest {
+
+    public static final String ALLOWLIST_PROPERTY = "elastic.apm.bootstrap_allowlist";
```

Comment:
[very minor] what about moving those constants to `BootstrapCheck` class directly and remove some duplication ? at least that allows to easily see when things are related to each other

Reply:
Much better, will do 👍